force-app/main/default/permissionsets/NKS_Tableau_Reporting.permissionset-meta.xml

```diff
@@ -30,4 +55,21 @@
         <object>LiveChatVisitor</object>
         <viewAllRecords>true</viewAllRecords>
     </objectPermissions>
+    <objectPermissions>
+        <allowCreate>false</allowCreate>
+        <allowDelete>false</allowDelete>
+        <allowEdit>false</allowEdit>
+        <allowRead>true</allowRead>
+        <modifyAllRecords>false</modifyAllRecords>
+        <object>NKS_Audit_Log__c</object>
+        <viewAllRecords>true</viewAllRecords>
```

Comment:
Jeg tror det er dumt å gi dem viewAll, det kan føre til problemer i fremtiden så vi burde unngå det i alle situasjoner vi kan egentlig

Reply:
Tror ikke dette vil skape problem, dette objektet er spesielt laget for innsikt over visninger (statistikk) og inneholder ikke noen sensitive data. View all gir dem mulighet til å se alle records i alle situasjoner noe som de har behov for. Jeg sjekket nå deres tilgang til andre objekter også og ser de har view all til alle de også